lib/group_domain_implementation/lib/src/accessors/firestore_course_member_accessor.dart

```diff
@@ -110,13 +123,13 @@ class FirestoreCourseMemberAccessor extends CourseMemberAccessor {
         if (docSnapshot.exists) {
           joinedUsersData = MemberData.create(
             id: docSnapshot.id,
-            role: docSnapshot.data()['powerLevel'] == 'owner'
+            role: docSnapshot.data()!['powerLevel'] == 'owner'
```

Comment:
Should we just assume that these attributes are present?

Reply:
Yes, since we're only assuming that there is data at all, which should always be true for a course.